src/BizHawk.Emulation.Common/Json/Array2DJsonConverter.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace BizHawk.Emulation.Common.Json
+{
+	// heavily inspired by https://stackoverflow.com/questions/66280645/how-can-i-serialize-a-double-2d-array-to-json-using-system-text-json
+	public class Array2DJsonConverter<T> : JsonConverter<T[,]?>
```

Comment:
Why delete my impl.? This looks similar, so a diff would be nice.

Reply:
There is no diff because the file was renamed in the same commit, but I did use your implementation as a basis. Of course I had to edit all function signatures and type names, so the difference was probably so great git didn't pick it up as the same file anymore. It should functionally be the same as your implementation though.

Reply:
Uhh yes I did comment on the wrong file.